### PR TITLE
[Compute RN: Maxwell Update 3 PCEE]: Release Notes Blurbs

### PR DESCRIPTION
**Pull request by divyabhushan-okta**
Compute RN: Maxwell Update 3 PCEE

### CODE REVIEW

File: cspm/rn/prisma-cloud-compute-release-information/features-introduced-in-compute-july-2023.adoc [hunk 1]
```diff
@@ -0,0 +1,157 @@
+[#id-july2023]
+== Features Introduced in July 2023
+
+Learn about the new Compute capabilities on Prisma™ Cloud Enterprise Edition (SaaS) in July 2023.
+
+The host, container, and serverless capabilities on the *Compute* tab are being upgraded starting on July 16, 2023. When upgraded, the version will be 30.03.xxx.
```
Comment (divyabhushan-okta) [inline cspm/rn/prisma-cloud-compute-release-information/features-introduced-in-compute-july-2023.adoc]: Build number to be added here > `30.03.xxx`

File: cspm/rn/prisma-cloud-compute-release-information/prisma-cloud-compute-known-issues.adoc [hunk 2]
```diff
@@ -15,6 +15,14 @@ The list of addressed issues are not cumulative; only the issues that are fixed
 |Issue ID
 |Description
 
+//CWP-49610
+|tt:[Fixed in 30.03.xxx]
```
Comment (divyabhushan-okta) [inline cspm/rn/prisma-cloud-compute-release-information/prisma-cloud-compute-known-issues.adoc]: Build number to be updated here.

File: cspm/rn/prisma-cloud-compute-release-information/prisma-cloud-compute-release-information.adoc [hunk 3]
```diff
@@ -9,9 +9,8 @@ On Prisma Cloud Enterprise Edition, the *Compute* tab comprises the Cloud Worklo
 
 // This release only includes fixes that are listed in xref:prisma-cloud-compute-known-issues.adoc#id311f617e-5159-4b1b-8cfa-29183c6e4a74[Prisma Cloud Compute Known Issues].
 The current *Compute* version is 30.02.123.
-//It will be upgraded to 30.03.xxx on Jul 16, 2023.
-To see the new capabilities and changes planned in July 2023, see the xref:look-ahead-planned-updates-prisma-cloud-compute.adoc[Prisma Cloud Compute Look Ahead].
-
+It will be upgraded to 30.03.xxx on Jul 16, 2023.
```
Comment (divyabhushan-okta) [inline cspm/rn/prisma-cloud-compute-release-information/prisma-cloud-compute-release-information.adoc]: Build number to be updated here, post the QA exit letter.

File: cspm/rn/prisma-cloud-compute-release-information/features-introduced-in-compute-july-2023.adoc (hunk 1 above)
Comment (rcaballeromx) [reply cspm/rn/prisma-cloud-compute-release-information/features-introduced-in-compute-july-2023.adoc]: Done

File: cspm/rn/prisma-cloud-compute-release-information/prisma-cloud-compute-release-information.adoc (hunk 3 above)
Comment (rcaballeromx) [reply cspm/rn/prisma-cloud-compute-release-information/prisma-cloud-compute-release-information.adoc]: Done

File: cspm/rn/prisma-cloud-compute-release-information/prisma-cloud-compute-known-issues.adoc (hunk 2 above)
Comment (rcaballeromx) [reply cspm/rn/prisma-cloud-compute-release-information/prisma-cloud-compute-known-issues.adoc]: Done

**Review by Pubs-MV — APPROVED** [review]

**Review by Pubs-MV — APPROVED** [review]

**Pull request merged**